### PR TITLE
New: sirWest.AudioSwitch version 2.2.2

### PR DESCRIPTION
**Pull request by jeandeaual**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27990)

### CODE REVIEW

File: manifests/s/sirWest/AudioSwitch/2.2.2/sirWest.AudioSwitch.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://github.com/sirWest/AudioSwitch/releases/download/2.2.2/AudioSwitchSetup.2.2.2.0.exe
+  InstallerSha256: BA77E802692A9B35B401A76624F5280249F889707EA7030EC652BF6018947012
+  InstallerSwitches:
+    Custom: /SUPPRESSMSGBOXES /NORESTART
+    Silent: /VERYSILENT
+    SilentWithProgress: /SILENT
+  UpgradeBehavior: install
```
Comment (OfficialEsco) [inline manifests/s/sirWest/AudioSwitch/2.2.2/sirWest.AudioSwitch.installer.yaml:18]:
```suggestion
PackageVersion: 2.2.2.0
MinimumOSVersion: 10.0.0.0
ProductCode: 'AudioSwitch_is1'
InstallerSwitches:
  Custom: /NORESTART
Installers:
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: inno
  Scope: machine
  InstallerUrl: https://github.com/sirWest/AudioSwitch/releases/download/2.2.2/AudioSwitchSetup.2.2.2.0.exe
  InstallerSha256: BA77E802692A9B35B401A76624F5280249F889707EA7030EC652BF6018947012
  UpgradeBehavior: install
```

File: manifests/s/sirWest/AudioSwitch/2.2.2/sirWest.AudioSwitch.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Created using YamlCreate.ps1 v1.1.7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: sirWest.AudioSwitch
+PackageVersion: 2.2.2
```
Comment (OfficialEsco) [inline manifests/s/sirWest/AudioSwitch/2.2.2/sirWest.AudioSwitch.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.2.2.0
```

File: manifests/s/sirWest/AudioSwitch/2.2.2/sirWest.AudioSwitch.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.1.7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: sirWest.AudioSwitch
+PackageVersion: 2.2.2
```
Comment (OfficialEsco) [inline manifests/s/sirWest/AudioSwitch/2.2.2/sirWest.AudioSwitch.yaml:5]:
```suggestion
PackageVersion: 2.2.2.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image
Since the Publisher is missing in ARP we need to add the ProductCode
The version needs to be 2.2.2.0 like it reports in ARP
Cleaned up the InstallerSwitches since the InstallerType is Inno

**Comment by OfficialEsco** [comment]
Remember you also need to change the version folder name

**Comment by jeandeaual** [comment]
> 
> 
> image
> Since the Publisher is missing in ARP we need to add the ProductCode
> The version needs to be 2.2.2.0 like it reports in ARP
> Cleaned up the InstallerSwitches since the InstallerType is Inno

Thanks for the corrections!

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**